MinVerTests.Packages/Net5PlusFact.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using MinVerTests.Infra;
+using Xunit;
+
+namespace MinVerTests.Packages
+{
+    public sealed class Net5PlusFact : FactAttribute
+    {
+        public override string Skip
+        {
+            get => base.Skip ??
+                (
+                    (
+                        Sdk.Version != null &&
+                        (
+                            Sdk.Version.StartsWith("2.", StringComparison.Ordinal) ||
+                            Sdk.Version.StartsWith("3.", StringComparison.Ordinal)
+                        )
+                    )
+                    ? "Not .NET 5 or later"

```

Comment:
"Not .NET 5 or later" ? Does the meaning of the attribute not in the opposite?

Reply:
If the version starts with `2.` or `3.` then it is _not_ .NET 5 or later.